src/Symfony/Component/Notifier/composer.json

```diff
@@ -30,6 +30,7 @@
         "symfony/esendex-notifier": "<5.3",
         "symfony/firebase-notifier": "<5.3",
         "symfony/free-mobile-notifier": "<5.3",
+        "symfony/gitter-notifier": "<5.3",
```

Comment:
can be removed since the bridge doesn't exist in < 5.3

Reply:
Done.